backend/coins/eth/account.go

```diff
@@ -399,20 +407,13 @@ func (account *Account) update() error {
 		}
 	}
 
-	var balance *big.Int
+	pendingAmount := pendingTxsAmount(outgoingTransactionsData, account.coin.erc20Token != nil)
 	if account.coin.erc20Token != nil {
 		balance, err = account.coin.client.ERC20Balance(account.address.Address, account.coin.erc20Token)
```

Comment:
Quite confusing to take a `balance` arg and then overwrite it. I'd centralize this and move the erc20 balance preparation also to the backend loop, so it's easier to follow and understand.